.github/workflows/inspect.yaml

```diff
@@ -0,0 +1,23 @@
+name: Inspect [lint, test, gosec]
+
+on:
+  [workflow_dispatch, push]
+jobs:
+  go-inspect:
+    env:
+      GO111MODULE: on
+      GOLINT: on
+    name: Inspect packages
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: go1.14 linters
+        uses: AbsaOSS/golang-pipeline/go1.14/linter@master
+
+      - name: go1.14 test
+        uses: AbsaOSS/golang-pipeline/go1.14/test@master
```

Comment:
Shouldn't we replace `golang-pipeline` with golangci-lint from https://github.com/AbsaOSS/gopkg/pull/2/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R3 ?

Reply:
Good point @ytsarev. In such case, should I update https://github.com/AbsaOSS/golang-pipeline/tree/master/go1.14/linter a bit ?

Reply:
I know @somaritane has another solution candidate on the table, but temporarily we can add update pipeline.

Reply:
@kuritka sounds good!

Reply:
@ytsarev, I made some investigations and the question is if we shouldn't go with this one: https://golangci-lint.run/usage/install/#ci-installation

At the moment, I switched to `golangci` pipeline (viz current)

Reply:
> I know @somaritane has another solution candidate on the table, but temporarily we can upgrade pipeline.

Yes, the suggestion was to use https://github.com/nektos/act which allows to run GHA workflows locally, so we don't have to maintain two different pipelines, one for local execution, another for GHA.